### PR TITLE
Add buildomat jobs for select images

### PR DESCRIPTION
**Pull request by labbott**
It's useful to have some images that have gone through a `permslip sign`. Do so in a very limited fashion.

### CODE REVIEW

File: .github/buildomat/force-git-over-https.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+#
+override_urls=(
+    'git://github.com/'
+    'git@github.com:'
+    'ssh://github.com/'
+    'ssh://git@github.com/'
+    'git+ssh://git@github.com/'
+)
+for (( i = 0; i < ${#override_urls[@]}; i++ )); do
+	git config --add --global url.https://github.com/.insteadOf \
```
Comment (mkeeter) [inline .github/buildomat/force-git-over-https.sh:14]: Dumb question: why don't we need quotes around "https://github.com", i.e.

```
git config --add --global url."https://github.com/".insteadOf
```
?

Comment (labbott) [reply .github/buildomat/force-git-over-https.sh:14]: I uh borrowed this setup from omicron https://github.com/oxidecomputer/omicron/blob/main/tools/include/force-git-over-https.sh

File: .github/buildomat/permslip-setup.sh [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+$PERMSLIP_DIR/target/release/permslip-server import-ssh-key /tmp/id_p256.pub
+$PERMSLIP_DIR/target/release/permslip-server import-private-key "UNTRUSTED bart" rsa "$BART_KEY"
+$PERMSLIP_DIR/target/release/permslip-server start-server &
+
+sleep 5
+
+$PERMSLIP_DIR/target/release/permslip --url=http://localhost:41340 list-keys
+
+# Gimlet
```
Comment (mkeeter) [inline .github/buildomat/permslip-setup.sh:49]: Why are we generating the gimlet / sidecar / psc keys, if we're only signing with `UNTRUSTED SP`?

Comment (labbott) [reply .github/buildomat/permslip-setup.sh:49]: That was from some earlier work that never went forward. I'll remove this

**Review by mkeeter — APPROVED** [review]

**Pull request merged**